soco/music_library.py

```diff
@@ -597,3 +598,49 @@ def album_artist_display_option(self):
         """
         result = self.contentDirectory.GetAlbumArtistDisplayOption()
         return result['AlbumArtistDisplayOption']
+
+    def list_library_shares(self):
+        """Return a list of the music library shares.
+
+        Returns:
+            list: The music library shares, which are strings of the form
+                ``'//hostname_or_IP/share_path'``.
+        """
+        response = self.contentDirectory.Browse([
+            ('ObjectID', 'S:'),
+            ('BrowseFlag', 'BrowseDirectChildren'),
+            ('Filter', '*'),
+            ('StartingIndex', '0'),
+            ('RequestedCount', '100'),
+            ('SortCriteria', '')
+        ])
+        shares = []
+        matches = response['TotalMatches']
+        # Zero matches
+        if matches == '0':
+            return shares
+        xml_dict = xmltodict.parse(response['Result'])
+        unpacked = xml_dict['DIDL-Lite']['container']
+        # One match
+        if matches == '1':
+            shares.append(unpacked['dc:title'])
+            return shares
+        # Otherwise it's multiple matches
+        for share in unpacked:
+            shares.append(share['dc:title'])
+        return shares
+
+    def delete_library_share(self, share_name):
+        """Delete a music library share. This is a 'fire and forget' operation
```

Comment:
You really don't need the "fire and forget" warning, and if you choose to keep it, I would suggest a neutral metaphor ;)

Reply:
Ok.